charts/prometheus-rabbitmq-exporter/templates/secret.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if or (and (.Values.rabbitmq.password) (not .Values.rabbitmq.existingPasswordSecret)) (and (.Values.rabbitmq.user) (not .Values.rabbitmq.existingUserSecret)) }}
```

Comment:
If `.Values.rabbitmq.password` is set but not `.Values.rabbitmq.existingPasswordSecret`, secret will not be created and Pod will never start. We should have a solution. Maybe we should prevent the two being set at the same time, or we align the condition in the two files, or we simply "ignore" the latter like it's done before this PR.

Reply:
Hi!

Yes, you're right. If existing password secret is set, then we don't need to create the secret, as we'll be using the one provided in the values.

On that case, the secret where we're reading the value for the env is also the one provided, so I think the pod will be created and running.

This is an example of the env for the deployment on that case:

```yaml
          env:
            - name: RABBIT_PASSWORD
              valueFrom:
                secretKeyRef:
                  name: "existingsecret"
                  key: "password"
            - name: RABBIT_USER
              valueFrom:
                secretKeyRef:
                  name: "existingsecret"
                  key: "username"
            - name: RABBIT_URL
              value: my-url.com
```

Reply:
About the options you mentioned, I followed what it was done before. If the existing secret is given, the password or user is ignored and the secret used.

Reply:
I am sorry, the diff bugged me, you are right! To be sure for the future, I'll add a few tests in a next PR.